apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -366,114 +409,73 @@
           "default": 1
         },
         "advanceScorePerc": {
-          "$ref": "#/definitions/advanceScorePerc"
+          "description": "Minimum score percentage to unlock access to subsequent questions",
```

Comment:
Nit: I noticed that when schema definitions are used, the description is repeated both where the schema is referenced and in the schema itself. Is this just how `zod-to-json-schema` works?

Reply:
What would you prefer? Just on the schema reference or just on the schema? I can play with the options.

Reply:
I'm actually not sure what I prefer 😅 I suppose it doesn't really make a difference, and maybe it's nice to be able to see a description where the ref is used?

Reply:
Sure!